orchagent/qosorch.cpp

```diff
@@ -555,6 +556,7 @@ task_process_status QosOrch::handlePfcToQueueTable(Consumer& consumer)
 QosOrch::QosOrch(DBConnector *db, vector<string> &tableNames) : Orch(db, tableNames)
 {
     SWSS_LOG_ENTER();
+    initColorACL();
```

Comment:
Prefer to capitalize acronyms as single words. e.g. initColorAcl();
Same as initSystemACLTable -> initSystemAclTable, initACLEntryForECN -> initAclEntryForEcn.
https://google.github.io/styleguide/cppguide.html#Function_Names